prisma/schema.prisma

```diff
@@ -229,3 +231,28 @@ model DepositHead {
 
   @@map("deposit_head")
 }
+
+model MaspTransaction {
+  id               Int                 @id @default(autoincrement())
+  created_at       DateTime            @default(now()) @db.Timestamp(6)
+  updated_at       DateTime            @default(now()) @updatedAt @db.Timestamp(6)
+  transaction_hash String              @db.VarChar
+  block_hash       String              @db.VarChar
+  asset_identifier String              @db.VarChar
```

Comment:
I was guessing this identifier is a hash, so I put as string. Wondering if we want any other attributes of the assets/transactions and if we want to index on the identifier.

Reply:
I don't think we need to search MASP transactions based on a given asset yet, so I think we can omit for now.